appveyor.yml

```diff
@@ -11,5 +11,41 @@ skip_tags: true
 build:
   project: ReferenceAssemblyAnnotator.sln
   verbosity: minimal
+test_script:
+- git clean -dxf tests
+- msbuild /restore tests/MultiTFM /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=net35 /warnaserror /v:m
```

Comment:
@sharwell What would happen if this was `/p:TargetFramework=net35` and `<TargetFramework>$(TestFramework)</TargetFramework>` was removed from the csproj?

Reply:
No idea 🤷‍♂